src/core/addonManager.js

```diff
@@ -28,10 +28,12 @@ type FirefoxAddon = {|
 |};
 
 export type MozAddonManagerType = {|
+  abuseReportPanelEnabled: boolean,
```

Comment:
Isn't this relatively new? Maybe it should be marked as potentially `undefined` to be sure. WDYT?

Reply:
Good idea.